portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -64,11 +80,56 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     commentEntity.setProperty(COMMENT_TEXT, commentText);
     commentEntity.setProperty(TIME_STAMP, timeStamp);
 
+    // TODO: Remove print statement and store image URL in database.
+    System.out.println(getUploadedFileUrl(request, /* formInputElementName= */ "image").get());
+
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(commentEntity);
 
     response.sendRedirect(REDIRECT_URL);
   }
+
+  /** Returns a URL that points to the uploaded file, or null if the user didn't upload an image file. */
```

Comment:
This JavaDoc needs to be updated to mention Optional.empty() and not null.

Reply:
fixed